toolkit/scripts/toolchain/list_toolchain_specs.sh

```diff
@@ -6,12 +6,15 @@ TOOLCHAIN_BUILD_FILE=$1
 OUTPUT_FILE=$2
 
 # Extract the specs built from toolkit/scripts/toolchain/build_official_toolchain_rpms.sh
-# Each spec that is built will be on its own line in the following format:
+# Each spec that is built will be on its own line in the following formats:
 #
 # build_rpm_in_chroot_no_install foo
```

Comment:
maybe refer to "foo-spec" on this and the next line to differentiate from "qualified-foo"

Reply:
Fixed, thanks!